orb-ui/src/sound/mod.rs

```diff
@@ -314,10 +329,83 @@ impl Player for Jetson {
             tracing::error!("Sound file not found: {:?}", sound_type);
         }
     }
+
+    /// Finds the sound file path for the given sound type and sends it to the
+    /// sound player only if the sink is empty
+    fn try_queue(&self, sound_type: Type) -> Result<bool> {
+        return if let Ok(sink) = self.sink.lock() {
```

Comment:
instead of doing:
```rust
if let Ok(sink) = self.sink.lock() {
  sink.whatever();
  sink.foo()
} else {
  Err(eyre!("..."))
}
```

use [let .. else](https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html) instead to detect the error and return early, allowing the rest of the code to proceed unindented:

```rust
let Ok(sink) = self.sink.lock() else {
  bail!("...");
};

// Notice how we are no longer indented unecessarily.
sink.whatever();
sink.foo()
```

By using this, I think that basically all of the nested if statments in this function can be un-nested, making it *much* simpler to understand.